build.gradle

```diff
@@ -1,14 +1,103 @@
-group 'org.broadinstitute'
-version '0.1'
+import io.swagger.codegen.config.CodegenConfigurator
+import io.swagger.codegen.DefaultGenerator
+
+buildscript {
+    ext {
+        springBootVersion = '2.0.5.RELEASE'
+    }
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
+        classpath('io.swagger:swagger-codegen:2.3.1')
+    }
+}
+
 
 apply plugin: 'java'
+apply plugin: 'idea'
```

Comment:
Is there a way to specify this is for local development? We might not want this for when we share with external developers if they're not using IntelliJ.

Reply:
As I read it, the plugin adds gradle targets for IntelliJ users and adds some Idea module metadata that IntelliJ can use. It doesn't require that someone to use IntelliJ to work on the project.

I do not think it is required for proper functioning of the rest of the gradle we have.